src/test.py

```diff
@@ -0,0 +1,20 @@
+# third party
```

Comment:
Is this script working? Because it looks like the `ddpm` cannot be loaded in tests.

Also, please remove this file from the git, keep it for local tests only.

Reply:
I met this problem of failing to load the new plugin when I tested it for the first time as well. What I did was to use pip to reinstall the project again `pip install -U .`. Is this problem during the checking due to the same reason?

Reply:
See my comment below. The problem is related to Py3.7/3.8